src/main/java/jR_Project3/models/users/AbstractUser.java

```diff
@@ -0,0 +1,8 @@
+package jR_Project3.models.users;
+
+public abstract class AbstractUser {
```

Comment:
what's the purpose of creating AbstractUser

Reply:
Because in next time i will do registration and i will add "RegisteredUser", "UnregisteredUser".. and now i did abstraction for extension this entity